README.rst

```diff
@@ -24,6 +24,10 @@ The last stable release is available on github and can be installed with ``pip``
     $ git clone https://github.com/rqlite/pyrqlite.git
     $ pip install ./pyrqlite
 
+Ofter are released on PyPI too, able to be installed via the package named ``rqlite``::
+
+    $ pip install rqlite
```

Comment:
But before you pointed it to me: https://github.com/rqlite/pyrqlite/issues/5#issuecomment-271491015

```
$ pip search rqlite
rqlite     - Pure python client library for rqlite
```